pages/pipelines/build_timeouts.md

```diff
@@ -38,3 +38,6 @@ By default, jobs are canceled when not picked up for 30 days. This will cause th
 You can override the default by setting a shorter value in your organization's [**Pipeline Settings**](https://buildkite.com/organizations/~/pipeline-settings) page.
 
 Scheduled job limits should not be confused with [scheduled builds](/docs/pipelines/scheduled-builds). A scheduled build's jobs will still go through the [build states](/docs/pipelines/defining-steps#build-states), and the timeout will apply once its individual jobs are in the scheduled state waiting for agents.
+
+> 📘 Delays in expiring a job
+> A job's expiry process is run hourly at 5 minutes past. When the expiry process runs and the job's scheduled expiry was not over at that hour, it will only be expired until the next hour when the process is executed again.
```

Comment:
In [US English](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#english-flavor), the word 'expiration' is used instead of 'expiry'.

Since we use US English throughout our docs as part of our styleguide, then:

```suggestion
> A job's expiration process is run hourly at 5 minutes past. When the expiration process runs and the job's scheduled expiration was not over at that hour, it will only be expired until the next hour when the process is executed again.
```

Also, since I can't easily do this through this PR, could you please update the other 4 remaining 'expiry' references on this page to 'expiration' please?
There are no internal docs links to the 'Scheduled job expiry' heading, and therefore, changing this to 'Scheduled job expiration' should be fine.